network_wrangler/RoadwayNetwork.py

```diff
@@ -116,3 +117,42 @@ def write(self, filename: str, path: str = '.') -> None:
 
         shapes_file = os.path.join(path, filename + "_shape.geojson")
         self.shapes_df.to_file(shapes_file, driver='GeoJSON')
+
+
+
+    def apply_roadway_feauture_change(self, project_card: str) -> bool:
+        '''
+        Changes the road network according to the project card passed
+        
+        args:
+        project_card: the path to the project card
+        
+        returns:
+        bool: True if successful.
+        '''
+        #import pdb; pdb.set_trace()
+        with open(project_card, 'r') as card:
```

Comment:
Ideally the project card object would hold the project card info rather than reading it in again with RoadwayNetwork.

Reply:
so, you would want project card to be read in a separate method and to pass the project card info dict as an arg into the method `apply_roadway_feauture_change`?

Reply:
Never mind! I see it. We already have a class (`ProjectCard.py`) to read project cards. So, it makes sense to pass the project card object to the method `apply_roadway_feauture_change`.